.flowconfig

```diff
@@ -1,11 +1,16 @@
 [ignore]
+./vendor/
+./node_modules/
 
 [include]
-app/javascripts/packs
 
 [libs]
 
 [lints]
 
 [options]
 module.name_mapper.extension='scss' -> 'empty/object'
+module.name_mapper='^Dashboard\(.*\)$' -> '<PROJECT_ROOT>/app/javascript/src/Dashboard/\1'
```

Comment:
Is it really needed to declare this for every module?
Do you know an alternative @didierofrivia ?

Reply:
hmmm, I wouldn't be that sure. In Policies was already setup to read the `@flow` tag at the beginning of the file. Since flow by its config was already looking for this.

Reply:
so we broke this when we included absolute path... first would be good to look for a better work around.

Reply:
I honestly don't see the problem in adding this. It doesn't make the code base any more complex, it doesn't make it harder to read or to maintain. It's confined in Flow's config file, can be erased or changed at any time.. and doesn't affect the app in any way. It makes the config file a little more verbose but just like `.eslint` does.